sdk/nodejs/region.ts

```diff
@@ -12,57 +12,53 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import {Region} from ".";
+
+/** @deprecated The AFSouth1Region constant is deprecated, use Region.AFSouth1 instead. */
```

Comment:
Nit: Simplify these messages to be consistent with messages elsewhere?

```suggestion
/** @deprecated Use Region.AFSouth1 instead. */
```